### ISSUE TITLE
binaries are too hard to find

### ISSUE THREAD

**Issue by Spongman** [TigerVNC/tigervnc#704]
i just watched a tester given a link to 'tigervnc.org' search your site for 5 minutes trying & failing to find a binary to download.

you need to have a huge browser-platform-specific download button front-and-center on your homepage.

like this:

![image](https://user-images.githubusercontent.com/1088194/44810123-9db9ce00-ab85-11e8-934e-4968d8a12486.png)

for most people, everything else is noise.

**Comment by hdatma** [comment]
If you click on "releases", you are presented with "https://github.com/TigerVNC/tigervnc/releases", which is where the binaries are.

**Comment by Spongman** [comment]
yeah, this was the primary reason for my complaint. for inexperienced users this is a maze of incomprehensible jargon & difficult to find links, resulting in frustration and eventual failure.

i want to be able to tell my users to download and user tigervnc, but currently i cannot - your download page is unusable. if i want to suggest using tigervnc i am going to have to implement my own platform detection and provide user-friendly download links, and handle updating that soehow when new versions are released. this obviously doesn't scale. it would make much more sense -- and likely increase your user base -- if this was done by tigervnc.

here's another example:
![image](https://user-images.githubusercontent.com/1088194/44928720-8efe2300-ad0d-11e8-86e0-6960840d1ce3.png)

**Comment by jglathe** [comment]
It can't hurt.

**Comment by grandchild** [comment]
I agree completely with the issue, but wouldn't this be an issue for the
[tigervnc.github.io](https://github.com/TigerVNC/tigervnc.github.io) repo?

**Comment by lucize** [comment]
@CendioOssman also would be fine if the source code archive has tigervnc in it's name, not only the version, it can be a burden to find the file in a download folder if you have multiple files there and you don't remember what is what

**Comment by CendioOssman** [comment]
It does have a proper name actually. It's just the URL that's a bit abbreviated. (And I know some software doesn't sort this out properly, but unfortunately this is out of our control and GitHub doesn't seem too eager to fix it. :/)

**Comment by lucize** [comment]
I don't know how to do it, but as an example https://github.com/FRRouting/frr/releases they could even add the tar.xz files, I guess someone knows about this, but as I understand, only when a new version is released the new changes takes effect

**Comment by pm-pm** [comment]
As a good workaround please add the "Download Button" (aka Link to Download page) at the beginning of "README.rst" file directly after the "About TigerVNC" section. ;-)
Thanks.

**Comment by pm-pm** [comment]
As a good workaround please add the "Download Button" (aka Link to Download page) at the beginning of "README.rst" file directly after the "About TigerVNC" section. ;-)
Thanks.

**Comment by MrTKusa** [comment]
CendioOssman commented on Sep 24, 2018
Technically, but we can let the issue live here as people will probably look here first. As for the issue at hand, it would of course be nice with a better  home page. But I don't see enough of an issue to give it priority right  now. If someone would like to make a suggestion it would be welcome  though.
--
I know this id somewhat an old statement but I just wanted to tell you that this attitude causes people to look elsewhere for software.  I am/was interested in you product enough to say this.
If the ability to dl your program is not "enough of an issue to give it priority right  now.", then when exactly will it be??  This is OK if you do not want people using your product.  May I remind you that each new user is not only a free tester but is also a chance to get a lot more new users.  This is how major products become major.     Customer first.  
IMHO

MrTKusa
